package.json

```diff
@@ -92,7 +92,9 @@
     "fbjs/isomorphic-fetch": "^3.0.0",
     "eslint-plugin-import": "^2.25.2",
     "minimist": "^1.2.6",
-    "url-parse": "^1.5.8"
+    "url-parse": "^1.5.8",
+    "google-gax": "2.30.5",
```

Comment:
@vbudhram A bit sketchy, but the crux of this PR turned out to be wrangling package dependencies. Without this, `@google-cloud/firestore` package would mysteriously stop working after installing the `@google-cloud/opentelemetry-cloud-trace-exporter` libraries. To make matters worse, the issue only manifested in the CI. For a while I suspected it had something to do with how we restore cache packages, but even disabling this mechanism completely would still result in failed test runs.
 
After digging through a bunch of package dependencies and trying to wrap my head around the differences between versions, this is what ultimately seems to have done the trick. I am somewhat optimistic that upgrading the various @google-cloud packages to latest versions would help, but unfortunately this also caused tests to fail, so it is probably a task for a separate issue.